session.go

```diff
@@ -199,6 +223,32 @@ func SetKeyValueDeletedCallback(f KeyValueDeletedCallback) {
 	keyValueDeletedCallback = f
 }
 
+// SetBulkDataResponseReceiver register a callback function when it received request from a DataStream.
+func SetBulkDataResponseReceiver(streamID string, receiver BulkDataResponseReceiver) error {
```

Comment:
Why is this function exported, but resubscribeBulkDataResponse is not? Will the user of the SDK call it?